metricq_source_ipmi/source/main.py

```diff
@@ -141,7 +143,31 @@ async def try_fix_hosts(conf, hosts_to_fix):
             conf['hosts'][host]['number_of_trys'] = 0
             conf['active_hosts'].add(host)
 
+def long_running_loop(loop):
```

Comment:
Same as below, why calling a Task object a loop?

Reply:
These are tasks in the sense of Python `Task`-objects, but are called "loops" because they do the same thing over and over. Maybe calling them `long_running_tasks` would work too.